.github/workflows/formatting.yaml

```diff
@@ -23,4 +23,11 @@ jobs:
         with:
           clang-format-version: '13'
           check-path: '.'
-#          fallback-style: 'Mozilla' # optional
+
+  tags:
+    name: Tags Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: check warnings
```

Comment:
Why another `name` key ?

Reply:
Because if I remove it, github complains:

```
a step cannot have both the `uses` and `run` keys
```

:)